moban/engine.py

```diff
@@ -96,6 +98,8 @@ def _file_permissions_copy(self, template_file, output_file):
         utils.file_permissions_copy(true_template_file, output_file)
 
     def _apply_template(self, template, data, output):
+        self.jj2_environment.globals['__target__'] = output
```

Comment:
The docs say `The core component of Jinja is the Environment. It contains important shared variables like configuration, filters, tests, globals and others. Instances of this class may be modified if they are not shared and if no template was loaded so far. Modifications on environments after the first template was loaded will lead to surprising effects and undefined behavior.` 
We are definitely modifying it after loading a template, which is bad.

`This is the place where you can put variables and functions that should be available all the time.`   We don't want it to be available all the time but only for a specific template.